infra/scripts/ci/test-unit.sh

```diff
@@ -2,11 +2,7 @@
 set -euxo pipefail
 
 # Run code tests
+GIT_ROOT=$(git rev-parse --show-toplevel)
+INFRA_DIR="infra"
 
-DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
-ROOT="$DIR/../.."
-
-(
-  cd "$ROOT"
-  "$ROOT"/node_modules/.bin/jest
-)
+"${GIT_ROOT}/${INFRA_DIR}/node_modules/.bin/jest"
```

Comment:
Won't this run Jest in whatever directory you're calling the script from? Aren't you missing the `cd ...` here?
```suggestion
cd ${GIT_ROOT}/${INFRA_DIR}"
"${GIT_ROOT}/${INFRA_DIR}/node_modules/.bin/jest"
```

Reply:
If changes in this file are not necessary, consider reverting to minimize change and risk